rfc/rfc017-distributed-network-grpc-v2.md

```diff
@@ -293,15 +303,16 @@ The `XOR` value MUST still be calculated over all transaction references on the
 
 #### 6.2.4 Transaction Range Query
 
-If the IBLT from a `TransactionSet` message can be decoded, the local node MAY also send a `TransactionRangeQuery` message.
+If the IBLT from a `TransactionSet` message can be decoded but contains no missing transactions, the local node SHOULD send a `TransactionRangeQuery` message.
```

Comment:
because of the 1 conversation limit this is an OR?

Reply:
Yes, we only send a RangeQuery when there is no ListQuery